src/modules/counter-comments.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable no-unused-expressions */
+
```

Comment:
- [ ] Please do not disable the linter checks because it helps to write better code. Kindly remove this and fix the linter error. You can make a quick googling on how to fix this without disabling the linter.